src/plots/geo/layout_attributes.js

```diff
@@ -174,9 +174,31 @@ var attrs = module.exports = overrideAll({
             min: 0,
             dflt: 1,
             description: [
-                'Zooms in or out on the map view.',
-                'A scale of *1* corresponds to the largest zoom level',
-                'that fits the map\'s lon and lat ranges. '
+                'Initial zoom level of the map view.',
+                'A scale of *1* (100%) corresponds to the base zoom level',
+                'that fits the map\'s lon and lat ranges.'
+            ].join(' ')
+        },
+        minscale: {
+            valType: 'number',
+            role: 'info',
+            min: 0,
+            dflt: 0,
+            description: [
+                'Minimal zoom level of the map view.',
+                'A minScale of *0.5* (50%) corresponds to a zoom level',
```

Comment:
Please replace minScale with \`minscale\`. And the same for `maxscale`. Thanks!

Reply:
fixed with d486d744e7d6e2aee0746118c02167e62c518797